.github/pull_request_template.md

```diff
@@ -1,8 +1,7 @@
-[Optional] 
-Fixes #issue
+Closes #issue (optional)
 
-*<Context of what this change does, why it is needed and how it is accomplished>*
+_<Context of what this change does, why it is needed and how it is accomplished>_
```

Comment:
Why replace `*` with `_`? They are equivalent and from what I understand, `*` is the slightly preferred one with `_` being there to disambiguate and allow combining them like so:
`_**bold/italic**_` - _**bold/italic**_

Reply:
This was not intentional but done by the auto formatter (prettier) since this is what we use for the project, it would be a real hassle to keep fixing it every time.

Reply:
Cool! If prettier does it, then might as well use it as the standard.